### ISSUE TITLE
Mass Obsoletion: focus on branch 'disorder of development or morphogenesis' (MONDO:0021147)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6876]
branch:  'disorder of development or morphogenesis' (MONDO:0021147)

Terms to obsolete:

CURIE | obsoletion candidate label
-- | --
MONDO:0017961 | 46,XX disorder of gonadal development
MONDO:0020039 | 46,XX disorder of sex development induced by androgens excess
MONDO:0017963 | 46,XX disorder of sex development induced by endogenous maternal-derived androgen
MONDO:0017964 | 46,XX disorder of sex development induced by exogenous maternal-derived androgen
MONDO:0019593 | 46,XX disorder of sex development induced by fetal androgens excess
MONDO:0017962 | 46,XX disorder of sex development induced by fetoplacental androgens excess
MONDO:0019608 | 46,XX disorder of sex development induced by maternal-derived androgen
MONDO:0017966 | 46,XY disorder of gonadal development
MONDO:0019597 | 46,XY disorder of sex development due to isolated 17,20-lyase deficiency
MONDO:0017974 | 46,XY disorder of sex development induced by maternal-exposure to endocrine disruptors
MONDO:0017969 | 46,XY disorder of sex development of endocrine origin
MONDO:0019826 | abnormal origin or aberrant course of coronary artery
MONDO:0017240 | acrodysostosis with multiple hormone resistance
MONDO:0017422 | adactyly of hand
MONDO:0019816 | anomaly of the tricuspid subvalvular apparatus
MONDO:0020147 | anophthalmia-microphthalmia syndrome
MONDO:0020286 | aortic malformation
MONDO:0019822 | arterial duct anomaly
MONDO:0020293 | ascending aorta anomaly
MONDO:0019837 | atrial appendage anomaly
MONDO:0020294 | atrial defect and interatrial communication
MONDO:0020288 | atrioventricular valve anomaly
MONDO:0017670 | autosomal dominant diffuse mutilating palmoplantar keratoderma
MONDO:0020093 | autosomal dominant isolated diffuse palmoplantar keratoderma
MONDO:0020096 | autosomal recessive isolated diffuse palmoplantar keratoderma
MONDO:0016647 | autosomal recessive Stickler syndrome
MONDO:0015334 | branchial arch or oral-acral syndrome
MONDO:0044655 | c12orf65-related combined oxidative phosphorylation defect
MONDO:0017104 | central nervous system cystic malformation
MONDO:0015915 | cerebellar malformation
MONDO:0016054 | cerebral malformation
MONDO:0016170 | chronic polyradiculoneuropathy
MONDO:0016034 | cleft lip with or without cleft palate
MONDO:0017448 | congenital absence/hypoplasia of fingers excluding thumb
MONDO:0017447 | congenital absence/hypoplasia of thumb
MONDO:0019831 | congenital anomaly of the coronary sinus
MONDO:0018185 | congenital anomaly of the great veins
MONDO:0018771 | congenital anomaly of ventricular septum
MONDO:0017428 | congenital deformities of fingers
MONDO:0018292 | congenital disorder of glycosylation-related bone disorder
MONDO:0015182 | congenital enteropathy involving intestinal mucosa development
MONDO:0019054 | congenital limb malformation
MONDO:0017710 | congenital systemic veins anomaly
MONDO:0020375 | coralliform cataract
MONDO:0020018 | cranial malformation
MONDO:0020132 | cranial nerve and nuclear aplasia
MONDO:0020145 | developmental defect of the eye
MONDO:0020019 | digestive tract malformation
MONDO:0020392 | discrete fixed membranous subaortic stenosis
MONDO:0019619 | duplication of the esophagus
MONDO:0018454 | dysostosis of genetic origin
MONDO:0800075 | dysostosis with predominant vertebral with and without costal involvement
MONDO:0019513 | esophageal malformation
MONDO:0015385 | external auditory canal aplasia/hypoplasia
MONDO:0018758 | familial patent arterial duct
MONDO:0018262 | fetal anticonvulsant syndrome
MONDO:0019998 | gastroduodenal malformation
MONDO:0017114 | global cerebellar malformation
MONDO:0020284 | heart position anomaly
MONDO:0957024 | hereditary 46,XX disorder of sex development
MONDO:0957025 | hereditary 46,XY disorder of sex development
MONDO:0017131 | hereditary cardiac anomaly
MONDO:0015961 | hereditary head and neck malformation
MONDO:0018562 | hereditary otorhinolaryngological malformation
MONDO:0957009 | hereditary posterior fossa malformation
MONDO:0018187 | hereditary syndromic Pierre Robin syndrome
MONDO:0016229 | hereditary vascular anomaly
MONDO:0015497 | hypoglossia/aglossia
MONDO:0017049 | hypomyelination neuropathy-arthrogryposis syndrome
MONDO:0017420 | intercalary limb defects
MONDO:0019999 | intestinal malformation
MONDO:0018916 | isolated anorectal malformation
MONDO:0018114 | isolated brachycephaly
MONDO:0015337 | isolated craniosynostosis
MONDO:0017667 | isolated diffuse palmoplantar keratoderma
MONDO:0017673 | isolated focal palmoplantar keratoderma
MONDO:0016805 | isolated oxidative phosphorylation complex disorder
MONDO:0018113 | isolated plagiocephaly
MONDO:0016518 | isolated punctate palmoplantar keratoderma
MONDO:0018112 | isolated scaphocephaly
MONDO:0017429 | joint formation defects
MONDO:0036042 | KAT6B-related multiple congenital anomalies syndrome
MONDO:0034024 | kyphoscoliotic Ehlers-Danlos syndrome
MONDO:0015504 | larynx anomaly
MONDO:0015418 | lateral facial cleft
MONDO:0016815 | Leigh syndrome with leukodystrophy
MONDO:0016816 | Leigh syndrome with nephrotic syndrome
MONDO:0018731 | lethal multiple congenital anomalies/dysmorphic syndrome
MONDO:0020146 | major induction processes eye anomaly
MONDO:0020131 | malformation of the cerebellar hemispheres
MONDO:0020130 | malformation of the cerebellar vermis
MONDO:0017676 | marginal papular palmoplantar keratoderma
MONDO:0016678 | maternal disease-related embryofetopathy
MONDO:0016794 | maternally-inherited mitochondrial myopathy
MONDO:0015412 | median facial cleft
MONDO:0017950 | microcephalic primordial dwarfism
MONDO:0017090 | midline cerebral malformation
MONDO:0019758 | midline interhemispheric variant of holoprosencephaly
MONDO:0016797 | multiple mitochondrial DNA deletion syndrome
MONDO:0017059 | neural tube closure defect
MONDO:0015219 | non-syndromic central nervous system malformation
MONDO:0034671 | non-syndromic complex polydactyly
MONDO:0017430 | non-syndromic congenital joint dislocations
MONDO:0015217 | non-syndromic developmental defect of the eye
MONDO:0015207 | non-syndromic esophageal malformation
MONDO:0015209 | non-syndromic gastroduodenal malformation
MONDO:0015211 | non-syndromic intestinal malformation
MONDO:0015227 | non-syndromic limb malformation
MONDO:0017431 | non-syndromic limb overgrowth
MONDO:0019713 | non-syndromic limb reduction defect
MONDO:0019714 | non-syndromic polydactyly, syndactyly and/or hyperphalangy
MONDO:0034670 | non-syndromic postaxial polydactyly
MONDO:0034669 | non-syndromic preaxial polydactyly
MONDO:0019720 | non-syndromic renal or urinary tract malformation
MONDO:0015221 | non-syndromic respiratory or mediastinal malformation
MONDO:0017421 | non-syndromic terminal limb defects
MONDO:0015619 | non-syndromic urogenital tract malformation
MONDO:0015932 | non-syndromic urogenital tract malformation of female
MONDO:0015933 | non-syndromic urogenital tract malformation of male
MONDO:0015829 | non-syndromic uterovaginal malformation
MONDO:0015213 | non-syndromic visceral malformation
MONDO:0015503 | nose and cavum anomaly
MONDO:0015415 | oblique facial cleft
MONDO:0015482 | otomandibular dysplasia
MONDO:0015330 | overgrowth/obesity syndrome
MONDO:0015499 | paralytic facial malformation
MONDO:0015478 | paramedian facial cleft
MONDO:0019712 | patellar dysostosis
MONDO:0035162 | PIK3CA-related overgrowth syndrome
MONDO:0020133 | posterior fossa malformation
MONDO:0016589 | progressive cerebello-cerebral atrophy
MONDO:0020287 | pulmonary artery or pulmonary branch anomaly
MONDO:0018580 | PURA-related severe neonatal hypotonia-seizures-encephalopathy syndrome
MONDO:0020023 | respiratory or mediastinal malformation
MONDO:0016355 | semilobar holoprosencephaly
MONDO:0017218 | septopreoptic holoprosencephaly
MONDO:0017423 | split hand or/and split foot malformation
MONDO:0020148 | syndromic aniridia
MONDO:0015246 | syndromic anorectal malformation
MONDO:0034661 | syndromic biliary atresia
MONDO:0015338 | syndromic craniosynostosis
MONDO:0015208 | syndromic esophageal malformation
MONDO:0015210 | syndromic gastroduodenal malformation
MONDO:0016565 | syndromic genetic obesity
MONDO:0015212 | syndromic intestinal malformation
MONDO:0015222 | syndromic respiratory or mediastinal malformation
MONDO:0015620 | syndromic urogenital tract malformation
MONDO:0015214 | syndromic visceral malformation
MONDO:0015323 | teratogenic Pierre Robin syndrome
MONDO:0015505 | tracheal anomaly
MONDO:0020285 | transposition of the great arteries and conotruncal cardiac anomaly
MONDO:0015837 | Unicervical bicornuate uterus
MONDO:0016803 | unspecified inborn mitochondrial disorder
MONDO:0020020 | visceral malformation of the liver, biliary tract, pancreas or spleen
MONDO:0018222 | X-linked intellectual disability due to GRIA3 anomalies

**Comment by sabrinatoro** [comment]
The spreadsheet was re-run after mass obsoletion was done on the other branches. The updated spreadsheet is [here](https://docs.google.com/spreadsheets/d/1ClfYywf9fhVSmKZ3SMfdGSb2SxR83cN0cdqIoCLd91I/edit#gid=2113377687)

**Comment by sabrinatoro** [comment]
This branch is currently messy, and further review is necessary.
The terms seem to be used inconsistently: 

> 'disorder of development or morphogenesis' (MONDO:0021147)
> Any disease or disorder that disrupts the process development of an anatomical structure. Can be due to genetic or environmental causes. Typically happens during embryogenesis, but also includes post-embryonic development.
According to this definition, "any" abnormality that is congenital or found in newborns should be in this class. However, these abnormalities should be focused on an anatomical structure. 
I found in this branch several diseases for which the etiology is an abnormal anatomical structure. Should these diseases be part of this branch? (that would probably be most diseases)

> 'developmental defect during embryogenesis'
> A disease that has its basis in the disruption of embryonic morphogenesis.
This definition is very confusing. Does it mean "embryonic" or "in the fetus" or "before birth"? The diseases under this class include all of these. Therefore it brings the question "what is the difference between this class and the one above"? 

In summary, this branch absolutely needs to be reviewed independently of the mass obsoletion. When reviewing the spreadsheet I: 
- kept the diseases that were clearly "morphological anomalies" that happened during gestation (e.g. dysplasia, aplasia, clefs,...)
- I didn't keep any syndromes unless the phenotypic features were fully or mostly "morphological phenotypes".
Note: my thought process evolved as I was going through the spreadsheet, so there might be inconsistencies.

**Comment by katiermullen** [comment]
@sabrinatoro nice work! I recommended a few more terms stay in the branch on the basis that they can be considered abnormalities in anatomical structures that arose during development or morphogenesis. Feel free to take them or leave them. I agree that future review is needed to improve the consistency of the terms in this branch.

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
